src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -422,38 +440,16 @@ This means the following state is added by {cheri_base_ext_name}:
 (draw-box "V" {:span 1 :borders {}}) (draw-box "Metadata" {:span 32 :borders {}}) (draw-box "Address" {:span 32 :borders {}})
 ----
 
-.New CLEN CSRs added in {cheri_base_ext_name}
-[#base_csr_registers]
-[bytefield]
-----
-(defattrs :plain [:plain {:font-family "M+ 1p Fallback" :font-size 40}])
-(def row-height 100)
-(def row-header-fn nil)
-(def boxes-per-row 65)
-(draw-column-headers {:height 30 :font-size 25 :labels (concat ["" "2*XLEN (CLEN)"] (repeat 31 "") ["XLEN"] (repeat 30 "") ["0"])})
-
-(draw-box "" [:box-first {:span 1}]) (draw-box "utidc (metadata)" [:box-related {:span 32}]) (draw-box "utidc (address)" [:box-last {:span 32 }])
-(draw-box "V" {:span 1 :borders {}}) (draw-box "Metadata" {:span 32 :borders {}}) (draw-box "Address" {:span 32 :borders {}})
```

Comment:
I guess this diagram doesn't add too much value but was it intentional to remove it?

Reply:
it was a duplicate of the one in the utidc section